keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -407,14 +439,18 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+Rollout or rollout fail do not impact already running workloads, only impact the new workloads.
```

Comment:
This is a general additional comment. This is subtler. Rolling out this option would mean removing the flag from the kubelet. Depending on the hardware, this will prevent the kubelet to start. I don't think rollout concerns really apply to this specific feature.
If you have a a machine wiht 9+ NUMA nodes AND you want to use a topology manager policy which is not None, THEN you need this option.

Reply:
Thanks for the review. I will update the KEP to address your comments.